soc/arm/renesas_rcar/common/clock_rcar.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#ifndef ZEPHYR_SOC_ARM_RENESAS_RCAR_COMMON_CLOCK_SOC_H_
+#define ZEPHYR_SOC_ARM_RENESAS_RCAR_COMMON_CLOCK_SOC_H_
+
+#include <dt-bindings/clock/renesas/renesas-cpg-mssr.h>
+
+int soc_cpg_core_clock_endisable(uint32_t base_address, uint32_t module,
```

Comment:
Please add documentation about these functions which include the possible error code.

Reply:
just documented these methods !